src/main/java/com/example/moviemanager/service/tmdbmovieservice/configuration/ApplicationConfig.java

```diff
@@ -0,0 +1,15 @@
+package com.example.moviemanager.service.tmdbmovieservice.configuration;
```

Comment:
instead should be in com.example.moviemanager.configuration package since it is whole application config